pkg/tfgen/docs.go

```diff
@@ -1271,9 +1271,9 @@ func (g *Generator) convertExamples(docs string, path examplePath) (result strin
 	}
 
 	// This function is very expensive for large providers. Permit experimental disk-based caching if the user
-	// specifies the PULUMI_CONVERT_EXAMPES_CACHE_DIR environment variable, pointing to a folder for the cache.
+	// specifies the PULUMI_CONVERT_EXAMPLES_CACHE_DIR environment variable, pointing to a folder for the cache.
```

Comment:
Can we keep this correction? Also good example why atomic PRs are good, one PR per concern.

Reply:
We are keeping that correction (commit is https://github.com/pulumi/pulumi-terraform-bridge/pull/1744/commits/490e7f5eff00a69a6c407eaed2976087379f99c2).

Reply:
The previous correction was accidentally reverted by https://github.com/pulumi/pulumi-terraform-bridge/pull/1689.